pkg/daemon/daemon.go

```diff
@@ -77,6 +77,10 @@ type Daemon interface {
 	// will be returned.
 	Backup(instanceId string) (backupId string, err error)
 
+	// Restore restores the backup with the given ID. If run is true, the instance
```

Comment:
Expand this a bit more. Tell about what happens if the instance exists. What you put in the README is a good reference of what is missing here.

Reply:
Done https://github.com/NethermindEth/eigenlayer/pull/90/commits/2e0422e0db1ca113579f7e5622caaa531dfd3b34